src/particles/distribution/Waterbag.H

```diff
@@ -0,0 +1,76 @@
+/* Copyright 2021 Ji Qiang, Axel Huebl
+ *
+ * This file is part of ImpactX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_DISTRIBUTION_WATERBAG
+#define IMPACTX_DISTRIBUTION_WATERBAG
+
+#include <AMReX_Random.H>
+#include <AMReX_REAL.H>
+
+#include <cmath>
+
+
+namespace impactx
+{
+namespace distribution
+{
+    struct Waterbag
+    {
+        /** A Waterbag distribution
+         *
+         * return 1 particle coordinates sampling from a 6D Waterbag distribution
+         * with mean = 0, std = 1.
```

Comment:
Do you like to expose the mean and std deviation as parameters?
This can be done here and stored as member variable (similar to how we expose element parameters).

Reply:
yes.

Reply:
Ok, I'll then wait until you add those before we merge it in.